utils.js

```diff
@@ -1,31 +1,5 @@
-let rolesCache = null;
 
-async function loadRolesIntoMemory(pool) {
```

Comment:
I think it's more logical to keep `loadRolesIntoMemory` in utils file, since this is not something frequently used, but rather a _utility_ code called once for service optimization. can we put it back?